### PR TITLE
Fix webpack warnings

### PR DESCRIPTION
**Pull request by jessevanmuijden**

### CODE REVIEW

**Review by gkatrakazas — APPROVED** [review]

**Comment by pstamatop** [comment]
@jessevanmuijden There's a minor conflict here - we’ll leave it to you to resolve, and we're ready to merge once it's done. Thanks!

**Comment by jessevanmuijden** [comment]
@pstamatop ✅ . FYI: new `==` operators and unused were introduced in master after I opened my PR. Perhaps we can add a github action that enforces code style guidelines.

**Pull request merged**
